### PR TITLE
Replay based fuzzer simple

### PR DESCRIPTION
**Pull request by oxade**
## Description 

Replay based fuzzer using a simple transaction mutation scheme.

## Test Plan 

Manual, unit
---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: crates/sui-replay/src/types.rs [hunk 1]
```diff
@@ -21,7 +21,7 @@ use tracing::error;
 
 // These are very testnet specific
 pub(crate) const TESTNET_GENESIX_TX_DIGEST: &str = "Cgww1sn7XViCPSdDcAPmVcARueWuexJ8af8zD842Ff43";
-pub(crate) const SAFE_MODE_TX_1_DIGEST: &str = "AGBCaUGj4iGpGYyQvto9Bke1EwouY8LGMoTzzuPMx4and";
+pub(crate) const SAFE_MODE_TX_1_DIGEST: &str = "AGBCaUGj4iGpGYyQvto9Bke1EwouY8LGMoTzzuPMx4nd";
```
Comment (tzakian) [inline crates/sui-replay/src/types.rs:24]: Did you mean to change this?

**Review by tzakian — APPROVED** [review]
This is awesome!

File: crates/sui-replay/src/types.rs (hunk 1 above)
Comment (oxade) [reply crates/sui-replay/src/types.rs:24]: Yup! it was accidentally changed by @lxfind in a refactor PR.

Comment (lxfind) [reply crates/sui-replay/src/types.rs:24]: oops, my bad! spelling-check attack

**Pull request merged**